documentation/API_documentation/S2CVPN_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# SiteToCloud(S2C)  VPN API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an application developer or industry customer, I want to request (using my application server/backend service)cloud leased network connection service between siteA/cloudA and  siteB/cloudB from a Communication Service Provider (CSP), so that I can ensure better and more reliable quality for our end users at any required time with proper cost. |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User<br> **Actors:** Application service providers,  industry customer,  application developers.<br> **Scope:** Service Booking(check network availability level, resource availability and corresponding fees )   \- Create/Remove/Modify/Query  service instance (S2C VPN session resource) |
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.</li><li>The Customer:Administrator has onboarded the Customer:User to the platform.</li><li>The Customer either owns the device subscriptions or has agreed to the terms and conditions of the CSP for managing consent of subscription owners.</li><li>The means to get the access token are known to the Customer:User to ensure secure access of the API.|
```

Comment:
I'll suggest that we'd better have a flow diagraph to better elaborate the relationship of the roles here.